appcues/src/main/java/com/appcues/ui/ExperienceRenderer.kt

```diff
@@ -229,9 +229,8 @@ internal class ExperienceRenderer(override val scope: AppcuesScope) : AppcuesCom
         }
     }
 
-    suspend fun dismiss(renderContext: RenderContext, markComplete: Boolean, destroyed: Boolean): ResultOf<State, Error> {
-        return stateMachines.getOwner(renderContext)?.stateMachine?.handleAction(EndExperience(markComplete, destroyed))
-            ?: Failure(RenderContextNotActive(renderContext))
```

Comment:
Failure was not being used at all, RenderContextNotActive here doesnt make too much sense, we have other similar methods in this class that calls to handleAction, and neither returns this error